gems/sorbet-runtime/lib/types/types/type_variable.rb

```diff
@@ -5,15 +5,16 @@ module T::Types
   # Since we do type erasure at runtime, this just validates the variance and
   # provides some syntax for the static type checker
   class TypeVariable < Base
-    attr_reader :variance
+    attr_reader :variance, :fixed
 
     VALID_VARIANCES = [:in, :out, :invariant]
 
-    def initialize(variance)
+    def initialize(variance, fixed=nil)
       if !VALID_VARIANCES.include?(variance)
         raise TypeError.new("invalid variance #{variance}")
       end
       @variance = variance
+      @fixed = fixed
```

Comment:
It's intentional that these weren't stored before.
Let's discuss this in details, there are substantial irreversible implications of storing it and thus let's not donut lightly

Reply:
I figured - this is a hack to get it working, but we could find an alternative approach if we wanted to actually ship this